### PR TITLE
Victor Kalinkin html-forms (initial submission)

### PR DESCRIPTION
**Pull request by vkalinkin**
html forms coding form

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <form action="">
+      <h1>Coding Form!</h1>
+      <div>Email</div>
```
Comment (scbowler) [inline html-forms/index.html:9]: Make sure to use `<label>` elements for the text related to your inputs. The label should have a `for` attribute that matches the `id` of the related input.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      <input type="text">
+
+
+      <div>Why is coding cool?</div>
+      <select name= "coding-reasons">
```
Comment (scbowler) [inline html-forms/index.html:14]: Use label with `for` attribute that matches the `id` of your `select`

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+      <div>Favorite Language (choose one)</div>
+      <div>
+        <input type="radio" name="language" id="html" value="html">
+        <label for="html">HTML</label>
+        <input type="radio" name="language" id="css" value="css">
+        <label for="css">CSS</label>
+        <input type="radio" name="language" id="js" value="js">
+        <label for="js">Javascript</label>
+      </div>
```
Comment (scbowler) [inline html-forms/index.html:30]: This part is perfect!

**Review by scbowler — CHANGES_REQUESTED** [review]
Few changes needed. Let me know if you have any questions!

**Comment by vkalinkin** [comment]
updated labels

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**